src/seedu/addressbook/parser/Parser.java

```diff
@@ -88,4 +88,152 @@ public static Command parseCommand(String userInput) {
             default:
                 return new HelpCommand();
         }
-    }
\ No newline at end of file
+    }
+
+    /**
+     * Parses arguments in the context of the add person command.
+     *
+     * @param args full command args string
+     * @return the prepared command
+     */
+    private static Command prepareAdd(String args){
```

Comment:
Good job on making these private methods static as well. As `parseCommand` was made static, any methods that it depends on has to be made static as well.